content/blog/أسس-تستضيف-خِدْمَات-تهتم-بالخصوصية-ال/index.ar.md

```
@@ -25,7 +25,7 @@ lang: "ar"
 
 ## قائمة الخِدْمَات المتوفرة
 
-![](service-annoucement-1024x576-3e4ff321-2.jpg)
+![](service-annoucement-1024x576-3e4ff321.jpg)
```

markdownlint-cli2 (0.22.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)